test/remoting/issue4007-reload-lost-app-window/test.py

```diff
@@ -0,0 +1,34 @@
+import time
+import os
+import platform
+import sys
+
+if platform.system() === 'Darwin':
+    print 'Skipped test for Mac due to a blocking issue #4165'
```

Comment:
Preferring not to fail any test is not the right thing to do here. The problem should not be covered.